js/components/user-bar.js

```diff
@@ -10,7 +10,7 @@ define([
 	'services/JobDetailsService',
 	'services/MomentAPI',
 	'lodash',
-	'less!./user-bar.less'
+	'less!components/user-bar.less'
```

Comment:
I don't understand the absolute path here. The user-bar.less file belongs to the user-bar module.

Reply:
This should have worked with the ./ path to the .less file.